src/controllers/update/overseas.entity.update.details.controller.ts

```diff
@@ -30,6 +31,8 @@ export const get = async (req: Request, res: Response, next: NextFunction) => {
 
     await fetchOverseasEntityEmailAddress(appData, req, session);
 
+    await fetchBeneficialOwnersPrivateData(appData, req);
```

Comment:
This needs to be before the email fetch. Should have used my branch instead of copying everything.

Reply:
got a bit messy when I tried it that way, will change these around.